Containerfile.redis

```diff
@@ -1,33 +1,17 @@
 # Use Red Hat Universal Base Image (UBI) 9 as the base image
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-# Install necessary dependencies (gcc, make, etc.) for building Redis from source
-RUN microdnf install -y gcc make wget tar gzip && \
-    microdnf clean all
+# Install necessary dependencies for downloading and installing Redis
+RUN microdnf install -y wget yum-utils logrotate systemd && \
```

Comment:
Why are we installing logrotate? Redis logs should be going into stdout/stderr.

Reply:
Yes. But redis itself requires logrotate as a dependency. If we try to build this image without it, we will run into this error
```
error: Failed dependencies:
	logrotate is needed by redis-6.2.7-1.el9.x86_64
```